docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -18,4 +19,15 @@ Person *--> Name
 Person *--> Phone
 Person *--> Email
 Person *--> Address
+Person *--> "0..1" Matric
+Person *--> "0..1" Studio
+Person *--> "0..1" Reflection
+
+Name -[hidden]right-> Phone
+Phone -[hidden]right-> Address
+Address -[hidden]right-> Email
+Email -[hidden]down-> Matric
```

Comment:
Good that you made the boxes go downward. Would be one long line otherwise

Reply:
This update is very insightful, as the symbolism of the Matric going downwards whereas the rest of the fields going right leaves much room for thought.